app/src/pages/playground/fetchPlaygroundPrompt.ts

```diff
@@ -124,7 +240,26 @@ export const instanceToPromptVersion = (instance: PlaygroundInstance) => {
     tools: instance.tools.map((t) => ({
       definition: t.definition,
     })),
-    // @TODO(apowell): Add description and invocationParameters
+    outputSchema:
+      instance.model.invocationParameters
+        .filter(
+          (i) =>
+            i.canonicalName === RESPONSE_FORMAT_PARAM_CANONICAL_NAME ||
+            i.invocationName === RESPONSE_FORMAT_PARAM_NAME
+        )
```

Comment:
nit - I don't really like using single character variables. It helps the reader if it's something like `invocationParam`

Reply:
Fixed